packages/markers/src/browser/problem/problem-tree-model.ts

```diff
@@ -4,9 +4,9 @@
 * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
 */
-import { ProblemMarker, ProblemManager } from './problem-marker';
-import { MarkerNode, MarkerTree, MarkerOptions } from '../marker-tree';
-import { MarkerTreeModel, MarkerTreeServices } from '../marker-tree-model';
+import { ProblemMarker, ProblemManager } from '../../common/problem-marker';
+import { MarkerNode, MarkerTree, MarkerOptions } from '../../common/marker-tree';
```

Comment:
I think `marker-tree` and `marker-tree-model` should stay in the browser. They are useless in the backend.